torch_geometric/nn/models/edge_bank_predictor.py

```diff
@@ -0,0 +1,161 @@
+"""
+EdgeBank is a simple strong baseline for dynamic link prediction
+it predicts the existence of edges based on their history of occurrence
+
+Reference:
+    - https://github.com/fpour/DGB/tree/main
+"""
+
+import warnings
+
+import torch
+
+
+class EdgeBankPredictor(object):
+    def __init__(
+        self,
+        src: torch.tensor,
+        dst: torch.tensor,
+        ts: torch.tensor,
```

Comment:
I think it is kind of bad practice to initialize a model with input data. I would prefer if users need to call `update` explicitly after initializing.